### PR TITLE
Added documentation for "Goto" feature.

### LINKED ISSUE #4452: Add documentation (and screen snippet) of new "Go to page" feature
We forgot to close an old issue and update the documentation for the new "Go to page / Jump to page" feature kindly done by @antoine2711 

### Current Results
There is no documentation for the new "Go to page - Page number" input box feature in the "Grid" section of docs - https://docs.openrefine.org/manual/running#the-grid

### Expected Behavior
Have a paragraph that describes:
- Changing numbers to go to a page
- Clicking in number input box and then using arrow keys UP/DOWN on keyboard can also advance/reverse page numbers
- Include a small screen snippet of the grid header area or better yet, the entire grid (`div#right-panel` in OpenRefine)

### Versions
 - Operating System: all
 - Browser Version: all
 - JRE or JDK Version: n/a
 - OpenRefine: 3.5+

### Additional context
As designed, discussed, tested in Issue #2638 and PR #2639

### PR DESCRIPTION
**Pull request by IMPranshu**
Fixes #4452 

Changes proposed in this pull request:
- Added a new section under Sort and View.''
- 4 new images/snippets of the new feature.

### CODE REVIEW

File: docs/versioned_docs/version-3.5/manual/sortview.md [hunk 1]
```diff
@@ -33,3 +33,24 @@ You can control what data you view in the grid. On each column, you will see a <
 You can find, under <span class="menuItems">All</span> → <span class="menuItems">View</span>, the option to show and hide [“null” values](exploring#data-types). A small grey “null” will appear in each applicable cell. Remember that a null cell is not the same thing as an empty cell. 
 
 ![A screenshot of what a null value looks like.](/img/null.png)
+
+## Goto {#goto}
```
Comment (thadguidry) [inline docs/versioned_docs/version-3.5/manual/sortview.md:37]:
```suggestion
## Page navigation {#pagenavigation}
```

File: docs/versioned_docs/version-3.5/manual/sortview.md [hunk 2]
```diff
@@ -33,3 +33,24 @@ You can control what data you view in the grid. On each column, you will see a <
...
+
+You can directly go to any page within your project.You could do a number of things here:
```
Comment (thadguidry) [inline docs/versioned_docs/version-3.5/manual/sortview.md:39]: Missing space.
```suggestion
You can directly go to any page within your project. You could do a number of things here:
```

File: docs/versioned_docs/version-3.5/manual/sortview.md [hunk 3]
```diff
@@ -33,3 +33,24 @@ You can control what data you view in the grid. On each column, you will see a <
...
+
+1. Select the the Row range (Rows per page) display.
```
Comment (thadguidry) [inline docs/versioned_docs/version-3.5/manual/sortview.md:41]:
```suggestion
1. Select the Row range (Rows per page) display.
```

File: docs/versioned_docs/version-3.5/manual/sortview.md [hunk 4]
```diff
@@ -33,3 +33,24 @@ You can control what data you view in the grid. On each column, you will see a <
...
+
+   ![A screenshot of the Total rows of the Projects.](/img/total-rows.png)
+
+2. The page range changes as "Rows Per Page" changes
+
+   ![A screenshot of the Page with 10 Rows Per Page selected.](/img/page10.png)
+   
+   Here you can see that 50 rows per page is been selected which changes the total number of pages in the project.
```
Comment (thadguidry) [inline docs/versioned_docs/version-3.5/manual/sortview.md:49]:
```suggestion
   Here you can see that 50 rows per page has been selected which changes the total number of pages displayed.
```

File: docs/versioned_docs/version-3.5/manual/sortview.md [hunk 5]
```diff
@@ -33,3 +33,24 @@ You can control what data you view in the grid. On each column, you will see a <
...
+   
+   ![A screenshot of the Page with 50 Rows Per Page selected.](/img/page50.png)
+
+3. You can directly enter any page number(within the range) to get the Rows of that page.
```
Comment (thadguidry) [inline docs/versioned_docs/version-3.5/manual/sortview.md:53]:
```suggestion
3. You can directly enter any page number(within the range) to get the Rows of that page.  Additionally, you can click inside the page number box and then use your keyboard up/down arrow keys to advance/reverse by 1 page.
```

**Review by thadguidry — CHANGES_REQUESTED** [review]

File: docs/versioned_docs/version-3.5/manual/sortview.md (hunk 1 above)
Comment (wetneb) [reply docs/versioned_docs/version-3.5/manual/sortview.md:37]: Yes. To be consistent with the rest, I would even propose `#page-navigation` as a hash key.

**Review by wetneb — COMMENTED** [review]
Thanks! I think we could do with maximum one screenshot: this feature is really self-explanatory, it does not justify a lot of megabytes of images. I would also keep the text relatively small.

**Comment by IMPranshu** [comment]
@wetneb I did those changes you asked for. Any other changes are required?

**Comment by wetneb** [comment]
I would simplify the text even further, with something like this:

> You can go directly to any page by changing the page number on the right-hand side. Using the up and down arrow keys in this input lets you go to the next and previous pages. You can also change the number of rows or records per page on the left-hand side of the menu bar.

What do you think of it? Is any information missing from it?

**Comment by IMPranshu** [comment]
> I would simplify the text even further, with something like this:
> 
> > You can go directly to any page by changing the page number on the right-hand side. Using the up and down arrow keys in this input lets you go to the next and previous pages. You can also change the number of rows or records per page on the left-hand side of the menu bar.
> 
> What do you think of it? Is any information missing from it?

No, I think everything that needs to be covered is pretty much in it. Should I go ahed and make the commit?

**Comment by wetneb** [comment]
That would be amazing!

**Comment by IMPranshu** [comment]
I have added the proposed changes, please have a look.

File: docs/versioned_docs/version-3.5/manual/sortview.md [hunk 6]
```diff
@@ -33,3 +33,9 @@ You can control what data you view in the grid. On each column, you will see a <
 You can find, under <span class="menuItems">All</span> → <span class="menuItems">View</span>, the option to show and hide [“null” values](exploring#data-types). A small grey “null” will appear in each applicable cell. Remember that a null cell is not the same thing as an empty cell. 
 
 ![A screenshot of what a null value looks like.](/img/null.png)
+
+## Page navigation {#page-navigation}
+
+You can go directly to any page by changing the page number on the right-hand side. Using the up and down arrow keys in this input lets you go to the next and previous pages. You can also change the number of rows or records per page on the left-hand side of the menu bar.
```
Comment (thadguidry) [inline docs/versioned_docs/version-3.5/manual/sortview.md:39]:
```suggestion
You can go directly to any page by changing the page number on the right-hand side. Using the up and down arrow keys in this input lets you go to the next and previous pages. You can also change the number of rows or records per page on the left-hand side of the view header bar.
```

**Review by thadguidry — CHANGES_REQUESTED** [review]

**Review by thadguidry — APPROVED** [review]

**Comment by thadguidry** [comment]
@wetneb Looks good to me now.  I think ready for merge.

**Comment by IMPranshu** [comment]
@thadguidry thank you very much, sir. It wouldn't have been possible without your and @wetneb sir's help. Thanks

**Pull request merged**